### DISCUSSION TITLE
Xaml specified C++ class member variables

### DISCUSSION THREAD

**Discussion by bongvisu** [microsoft/WindowsAppSDK#3191] [General]
As far as I know, it's not recommended to name an identifier with a leading underscore, but I see
WinUI C++/WinRT code-behind class variable names start with a leading underscore.
Wouldn't it rather be better to use a trailing underscore?

**Comment by wjk** [comment]
[According to the inestimable Raymond Chen](https://devblogs.microsoft.com/oldnewthing/20230109-00/?p=107685), leading underscores are permitted in C++ identifiers _if the next letter is lowercase_. (Also, I neither quite got the practice of putting trailing underscores on instance variables. To me it looks really weird and sticks out like a sore thumb when read. I would rather use no underscore at all.)
